lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * dummylayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
+  var nodes = graph.nodes;
```

Comment:
You looked at Salvador's data model classes, right? Graph has no property nodes. It has a getter instead.